runbotics/runbotics-orchestrator-ui/src/src-app/components/tables/HistoryTable/HistoryTable.tsx

```diff
@@ -137,6 +137,10 @@ const HistoryTable = forwardRef<any, HistoryTableProps>(({ botId, processId, sx,
         }
     };
 
+    useEffect(() => {
+        replaceQueryParams({ page, pageSize, tab, id, instanceId });
+    }, [instanceId])
```

Comment:
you dont need a useEffect here. just run in whenever the user clicks on the history record